hack/ci/rbac_utils.sh

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+function rbac_cleanup {
+	kubectl delete clusterrolebinding etcd-operator
+	kubectl delete clusterrole etcd-operator
+}
+
+function rbac_setup() {
+    # Create ClusterRole
+    cat <<EOF | kubectl create -f -
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
```

Comment:
The lack of indentation is a little ugly but necessary since it's a heredoc and indenting would mess up the yaml file structure.